job/storage/mongo/mongo_test.go

```diff
@@ -0,0 +1,106 @@
+package mongo
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ajvb/kala/job"
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func NewTestDb(t *testing.T) *DB {
+	collection = "test"
```

Comment:
Should we name this something more kala-specific?

Reply:
@ajvb we do have the database as "kala". This is to make sure we don't cause any issues with running tests on the default "jobs" collection.

Reply:
oh gotcha, ok awesome.